helm/idol-nifi/values.yaml

```diff
@@ -222,10 +222,20 @@ nifi:
   # -- optional nifi.sensitive.props.key value (see https://nifi.apache.org/docs/nifi-docs/html/administration-guide.html#nifi_sensitive_props_key)
   # Setting this value is recommended. If it is not set, it will default to a generated value
   sensitivePropsKey: ""
+  # -- optional hostname of an existing nifi registry instance.
+  # Defaults to the created registry instance when nifiRegistry.enabled=true
+  registryHosts: ""
```

Comment:
Feels a bit odd that this is defined as plural but is singular string value but I guess that's from the environment variable name it maps down to?

Reply:
It is only helm/idol-nifi/resources/nifi-import-flow.sh that doesn't handle this containing multiple values, potentially that script should be updated to support multiple hosts, and then the description here could be made plural. Alternatively, if there is no likelihood of ever requiring multiple registries, then this property could be made singular. Preference?